src/components/account/DeleteAccount.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import validateAccountId from '../../utils/validateAccountId'
+
+const DeleteAccount = () => {
+  const [accountId, setAccountId] = useState('')
+  const [isFetchingData, setIsFetchingData] = useState(false)
+  const [isSuccessful, setIsSuccessful] = useState(false)
+
+  //errors
+  const [error, setError] = useState(false)
+  const [errorMessage, setErrorMessage] = useState<string | null>(null)
+
+  return (
+    <div className="DeleteAccount">
+      <h3>Delete an account</h3>
+      <input
+        type="text"
+        value={accountId}
+        placeholder="9e28507a-632f-44a6-99a7-98695cf2adcf"
+        onChange={(e) => setAccountId(e.target.value)}
+      ></input>
+      <button
+        className="deleteAccountBtn"
+        onClick={() => {
+          const validateInput = validateAccountId(accountId)
+          if (validateInput) {
+            setErrorMessage(validateInput)
+            setError(true)
+            setIsSuccessful(false)
```

Comment:
with the previous suggestion, you now can handle your request status through one setState call
```suggestion
            setRequestStatus('error')
```